extensions/wikia/ArticleVideo/bower_components/html5-skin/js/controller.js

```diff
@@ -0,0 +1,1496 @@
+/********************************************************************
+ CONTROLLER
+ *********************************************************************/
+var React = require('react'),
+    ReactDOM = require('react-dom'),
+    Utils = require('./components/utils'),
+    CONSTANTS = require('./constants/constants'),
+    AccessibilityControls = require('./components/accessibilityControls'),
+    Fullscreen = require('screenfull'),
+    Skin = require('./skin'),
+    SkinJSON = require('../config/skin'),
+    Bulk = require('bulk-require'),
+    Localization = Bulk('./config', ['languageFiles/*.json']);
+
+OO.plugin("Html5Skin", function (OO, _, $, W) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the unused function parameter "W". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AUnusedFunctionArgument)